src/SprykerEco/Zed/Unzer/Dependency/UnzerToQuoteClientBridge.php

```diff
@@ -27,7 +27,7 @@ public function __construct($quoteClient)
     /**
      * @return \Generated\Shared\Transfer\QuoteTransfer
      */
-    public function getQuote()
+    public function getQuote(): QuoteTransfer
```

Comment:
Please do not change method declarations in bridges

Reply:
Fixed